setup.rst

```diff
@@ -37,32 +37,36 @@ In other words, your new app is ready!
 Running your Symfony Application
 --------------------------------
 
-On production, you should use a web server like Nginx or Apache
-(see :doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
-But for development, it's convenient to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
+On production, you should use a web server like Nginx or Apache (see
+:doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
+But for development, it's better to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
 
-.. note::
+This local server provides support for HTTP/2, TLS/SSL, automatic generation of
+security certificates and many other features. It works with any PHP application,
+not only Symfony projects, so it's a very useful development tool.
 
-    If you want to use a virtual machine (VM) with Vagrant, check out
-    :doc:`Homestead </setup/homestead>`.
-
-Move into your new project and start the server:
+Once installed, move into your new project and start the local web server:
 
 .. code-block:: terminal
 
     $ cd my-project
     $ symfony server:start
```

Comment:
For symfony 4.3 docs, it might make sense to also mention the included webserverbundle way of starting the server (a version of this is mentioned a bit further down). So it could show the recommended `symfony server:start`, but also show `bin/console server:start` or `bin/console server:run` for environments that might not be able to install the symfony local server for some reason. (Purely hypothetical; I don't know if there's any place where PHP would be able to start a server but the user wouldn't be able to install the symfony local server. Perhaps other people have experience with that?)

Reply:
The leaders of the Symfony project decided to stop recommending the old WebServerBundle in docs (and also deprecate it and remove it in 5.0). We (the Docs Team) agree with that so we prefer to keep the old WebServerBundle out of the docs.

In addition to all the features of the local Symfony server, using it requires just one installation per system ... in contrast with the bundle, which requires adding it as a dependency of all your projects.

We understand your position because you are used to WebServerBundle, but we hope you also understand this decision to move things forward and improve DX while developing locally. Thanks!